README.md

```diff
@@ -37,12 +37,12 @@ Projects listed on this page are open source that Microsoft maintain or contribu
-|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: incubating <br> API: v1alpha1 and v1beta1 | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | <ul><li>[Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks)</ul><li>[Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine)</li><li>Azure Policy for Azure Arc connected clusters</li><ul> |
-|  | [Keyvault Flexvol](https://github.com/Azure/kubernetes-keyvault-flexvol) <br> ([releases](https://github.com/Azure/kubernetes-keyvault-flexvol/releases)) | Mount Azure Keyvault secrets into pods | Azure: incubation <br> API: N/A | [GitHub issues](https://github.com/Azure/kubernetes-keyvault-flexvol/issues) | [Use with AKS](https://docs.microsoft.com/en-us/azure/aks/developer-best-practices-pod-security#use-azure-key-vault-with-flexvol) |
+|  | [OPA Gatekeeper](https://github.com/open-policy-agent/gatekeeper) <br> ([releases](https://github.com/open-policy-agent/gatekeeper/releases)) | K8s native Open Policy Agent policy enforcement | CNCF: incubating <br> API: v1alpha1 and v1beta1 | [#kubernetes-policy](https://openpolicyagent.slack.com/archives/CDTN970AX) <br> [GitHub issues](https://github.com/open-policy-agent/gatekeeper/issues) | <ul><li>[Azure Policy for AKS](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/rego-for-aks)</ul><li>[Azure Policy for AKS Engine](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/aks-engine)</li><li>[Azure Policy for Azure Arc connected clusters](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/policy-for-kubernetes#install-azure-policy-add-on-for-azure-arc-enabled-kubernetes)</li><ul> |
 |  | [Secrets Store CSI Driver](http://sigs.k8s.io/secrets-store-csi-driver) <br> ([releases](https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases)) <br> [Builds](https://testgrid.k8s.io/sig-auth-secrets-store-csi-driver#secrets-store-csi-driver-e2e-vault-postsubmit) | Integrates secrets stores with Kubernetes via a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) volume | CNCF: N/A <br> API: v1alpha1 | [#sig-auth](https://kubernetes.slack.com/archives/C0EN96KUY) <br> [GitHub issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues) |  |
```

Comment:
Should we also update the slack channel for Secrets Store CSI Driver? Right now it's pointing to the `sig-auth` channel.